src/json_reporter.cc

```diff
@@ -167,12 +167,24 @@ bool JSONReporter::ReportContext(const Context& context) {
   }
   out << "],\n";
 
+#if defined(BENCHMARK_VERSION)
+  const char library_version[] = BENCHMARK_VERSION;
+#else
+  const char library_version[] = "hello, bazel!";
```

Comment:
I don't quite know what should be done for non-CMake.

Reply:
in bazel we could add a define to the `cc_library` that maps to `module_version()` i think: https://bazel.build/rules/lib/toplevel/native#module_version

Reply:
Yeah, i have no idea how to actually write that though. I've tried the obvious approach, but it may not work.

Reply:
Yes, i tried that first.